packages/core/src/Stores/portfolio-store.js

```diff
@@ -408,12 +420,35 @@ export default class PortfolioStore extends BaseStore {
 
         this.positions[i].is_loading = false;
 
-        if (
-            this.root_store.ui.is_mobile &&
-            getEndTime(contract_response) &&
-            window.location.pathname === routes.trade
```

Comment:
If we remove the condition to check the path, will it show a notification banner if the users navigate to dbot, cashier, or any other apps?

Reply:
Actually the condition of checking the pass for old Notification (from current production) is still here (line 426).
As for showing new notifications, here is a screen recording: seems that new notifications are not shown outside of DTrader-V2, but anyway I'll expand the qa-checklist

https://github.com/user-attachments/assets/3ceff3ea-3e7f-4be5-bcd8-4947d9270a16

Reply:
<img width="900" alt="Screenshot 2024-08-02 at 9 40 18 AM" src="https://github.com/user-attachments/assets/b165ddf7-91d5-4a7c-ae43-1bbe3192c7d4">